code/game/dna/dna2.dm

```diff
@@ -189,6 +190,40 @@ GLOBAL_LIST_EMPTY(bad_blocks)
 		val=rand(1, 2049)
 	SetUIValue(block, val, defer)
 
+//Get Tri State Block State
+/datum/dna/proc/GetUITriState(block)
+	if(block <= 0)
+		return
+	else
```

Comment:
Else not required here due to the early return at line 196

Reply:
Oh! Sweet!